src/modules/crystal.rs

```diff
@@ -0,0 +1,58 @@
+use ansi_term::Color;
+use std::process::Command;
+
+use super::{Context, Module};
+
+/// Creates a module with the current Crystal version
+///
+/// Will display the Crystal version if any of the following criteria are met:
+///     - Current directory contains a `.cr` file
+///     - Current directory contains a `shard.yml` file
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_crystal_project = context
+        .new_scan_dir()
+        .set_files(&["shard.yml"])
+        .set_extensions(&["cr"])
+        .scan();
+
+    if !is_crystal_project {
+        return None;
+    }
+
+    match get_crystal_version() {
+        Some(crystal_version) => {
+            const CRYSTAL_CHAR: &str = "💎 ";
```

Comment:
Hmmm....we're already using bold red with this symbol for [ruby](https://github.com/starship/starship/blob/master/src/modules/ruby.rs#L24). Is that going to get confusing?

Reply:
yeah, I thought so too, but I have no idea what to use instead.

Reply:
Maybe the "crystal ball"? 🔮